### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by robythedoct46** [petrleocompel/hikaxpro_hacs#49] [Q&A] [answered]
Thanks for your work hope you can add more devices in the integration.
Hikvision is increasing compatible devices and I would like to integrate into Home assistant. Thanks again

**Comment by petrleocompel** [comment]
Just list device, write model etc... Or provide debug json. And we can make it happen.

**Comment by cmore77** [comment]
Hi can you indicate what to fill for "Host"? Is it the ip address of the AX pro unit?
When adding the integration, there is a popup to  complete details. There is dead link behind the question mark icon...
:)

**Reply by petrleocompel** [reply]
Yes

**Reply by petrleocompel** [reply, marked as answer]
The link is "default link" from HomeAssistant - we do not have any documentation for `hikvision_axpro` on Home-assistant.io
https://www.home-assistant.io/integrations/hikvision_axpro 
You can see the link.. Currently I do not know if there is any possibility to change it...
